packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -285,6 +285,10 @@ export default class CodeMode extends Component<Signature> {
     return state;
   });
 
+  @action updateCodePath(newUrl: URL) {
```

Comment:
I think each child component can directly use `this.operatorModeStateService.updateCodePath(newUrl);`, so we don't need this function.

Reply:
ahh good catch! I think I was deciding between the 2 options. But ok ill put it in the child component